hedera-node/src/main/java/com/hedera/services/state/migration/MigrationRecordsManager.java

```diff
@@ -124,6 +143,28 @@ private TransactionBody.Builder synthStakingFundCreate(final long autoRenewPerio
 		return TransactionBody.newBuilder().setCryptoCreateAccount(txnBody);
 	}
 
+	private void publishContractFreeAutoRenewalRecords(final Instant now) {
+		MiscUtils.forEach(accounts.get(), (id, account) -> {
+			if (account.isSmartContract()) {
+				final var contractNum = id.toEntityId();
+				final var newExpiry = account.getExpiry();
+
+				final var receipt = new TxnReceipt();
+				receipt.setAccountId(contractNum);
```

Comment:
I just opened a [HIP PR](https://github.com/hashgraph/hedera-improvement-proposal/pull/449) on the record stream format for smart contract expirations; I proposed leaving out the receipt id since the target `ContractID` is going to be in the accompanying `ContractUpdateTransactionBody`.

Reply:
Removed !